### PR TITLE
Add FXIOS-6967 [v118] Create Settings Card

### PR DESCRIPTION
**Pull request by PARAIPAN9**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6967)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/15472)

## :bulb: Description

- Implement the Settings Card

https://github.com/mozilla-mobile/firefox-ios/assets/51127880/3db3bad4-db00-41bb-bdb4-696d3a1ecbe8

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -9874,6 +9876,7 @@
 				8CB19EB12A83C5280023F88E /* FakespotErrorCardView.swift */,
 				E16258EE2A83BE0800522742 /* FakespotLoadingView.swift */,
 				E1FC23F02A8629380089E14D /* ReliabilityCardView.swift */,
+				DF5D474F2A8A5B4400D6AE74 /* FakespotSettingsCardView.swift */,
```
[the remark this answers is not in the document]
Comment (PARAIPAN9) [reply Client.xcodeproj/project.pbxproj:9879]: It's already there

**Review by thatswinnie — COMMENTED** [review]
Please make sure that the switch is centered vertically to the switch label. It is really visible that it's not when using large dynamic type.
Simulator Screenshot - iPhone 14 Pro - 2023-08-15 at 15 57 07

File: Client/Frontend/Fakespot/FakespotViewController.swift [hunk 2]
```diff
@@ -122,23 +111,20 @@ class FakespotViewController: UIViewController, Themeable {
...
-        contentStackView.addArrangedSubview(errorCardView)
-        contentStackView.addArrangedSubview(reliabilityCardView)
-        contentStackView.addArrangedSubview(loadingView)
         scrollView.addSubview(contentStackView)
         [logoImageView, titleLabel].forEach(headerStackView.addArrangedSubview)
+        [errorCardView, reliabilityCardView, settingsCardView, loadingView].forEach(contentStackView.addArrangedSubview)
 
         NSLayoutConstraint.activate([
-            contentStackView.topAnchor.constraint(equalTo: scrollView.contentLayoutGuide.topAnchor,
+            contentStackView.topAnchor.constraint(equalTo: scrollView.topAnchor,
```
[the remark this answers is not in the document]
Comment (PARAIPAN9) [reply Client/Frontend/Fakespot/FakespotViewController.swift]: Yes I can revert to the previous changes, no problem, but we have to do something with the leading and trailing constraints, maybe to constraint them to the view's  safe area. Right now it doesn't look good in landscape at all.

[the remark this answers is not in the document]
Comment (thatswinnie) [reply Client/Frontend/Fakespot/FakespotViewController.swift]: This is getting fixed in [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/16162).

**Pull request closed without merging**